dev-tools/mage/target/common/notice.go

```diff
@@ -115,5 +115,21 @@ func Notice() (err error) {
 		return fmt.Errorf("failed to append file %s: %w", outfn, err)
 	}
 
+	// dos2unix NOTICE.txt
+	fmt.Printf(">> %s\n", "dos2unix NOTICE.txt")
```

Comment:
opening my comment here so it is clearly visible, I don't believe this step is necessary if you tune your git config properly and make it change CLRF endings to LF, which I believe it asks you to do so by default in windows. @intxgo any reason you think this isn't sufficient?

Reply:
This is a fix for the CI side of NOTICE.txt generation. The build machine in CI is producing NOTICE.txt with inconsistent line endings, so when I commit a file with all Unix-style it won't match. You can download the NOTICE.txt from my PR, it's clean all unix-style so should be green, agree?

Reply:
Sorry but on my end, I don't see what you say above @intxgo, as shown below, clearly you pushed a NOTICE.txt with CLRF endings at some point which you tried to fix with dos2unix step you introduced afterwards. 

<img width="916" alt="image" src="https://github.com/user-attachments/assets/c6d833ff-628d-45fc-89f8-e39f74635c8b" />

Something that could definitely help in our review is you to rewrite your commit history and initially keep only the commits that are necessary for the actual issue. If then the check-ci step fails because the CI machine produces a NOTICE.txt with CLRF ending you can do one extra commit with your dos2unix step

Reply:
please download the NOTICE.txt from main, and examine it exactly where the diff on my PR points you

![image](https://github.com/user-attachments/assets/5fb8d0b3-3e98-4d7b-9f52-0de0539aedad)

Reply:
then download the file from my branch and view/compare

Reply:
@intxgo AFAICT your PR affects only go-sysinfo, thus I would expect only this thing to be updated in the NOTICE.txt. Again please try to make this easier for the reviewers and tidy up your commit history, as is now it is not clear that the issue you are trying to fix in mage exists indeed. Do one commit with all the changes related to this issue only to make it clear that the CI machine is producing a wrong NOTICE.txt file and then push your dos2unix fix. I don't believe this is much of an ask here